src/Kafka/Settings/Settings.tsx

```diff
@@ -0,0 +1,148 @@
+import {
+  Alert,
+  AlertActionCloseButton,
+  AlertGroup,
+  AlertVariant,
+  Button,
+  Card,
+  CardBody,
+  Flex,
+  FlexItem,
+  Level,
+  LevelItem,
+  Modal,
+  ModalVariant,
+  Page,
+  PageSection,
+  Spinner,
+  Switch,
+  Title,
+} from "@patternfly/react-core";
+import type { AlertProps } from "@rhoas/app-services-ui-shared";
+import type { FunctionComponent } from "react";
+import { useTranslation } from "react-i18next";
+
+export type SettingsProps = {
+  showAlert: AlertProps[];
+  isChecked: boolean;
+  isModalOpen: boolean;
+  onChange: () => void;
+  onDisable: () => void;
+  onClose: () => void;
+  closeAlertAction: (key: string | undefined) => void;
+  isLoading: boolean;
+};
+
+export const Settings: FunctionComponent<SettingsProps> = ({
+  showAlert,
+  isChecked,
+  onChange,
+  isModalOpen,
+  onDisable,
+  onClose,
+  closeAlertAction,
+  isLoading,
+}) => {
+  const { t } = useTranslation("kafka");
+
+  return (
+    <>
+      <PageSection>
+        <Card>
+          <CardBody>{t("settings.kafka_instance_settings_label")}</CardBody>
+        </Card>
+      </PageSection>
+      <Page>
+        <PageSection isWidthLimited isCenterAligned>
+          <Card>
+            <CardBody>
+              <Level>
```

Comment:
The `Level` layout doesn't provide the modifiers needed to implement the layout we want for smaller viewports [shown here](https://docs.google.com/document/d/1hO_aMtRV75F48d4PV1j_LY1GMVXgFKwANeUwvJTzMoQ/edit?disco=AAAAavQirGA). In your implementation, the switch wraps underneath the text, but in that example, the switch is always on the right and only the text wraps.

I checked with one of the PF UI devs for suggestions, and this was his recommendation:
> I would just use a flex layout set to nowrap with the text flex child set to flex grow - or you could use a split layout with the text flex child set to "fill" (.pf-m-fill or the isFilled react prop)